app/api/getPoints/route.ts

```diff
@@ -1,21 +1,28 @@
 import { stack } from '@/lib/stack/client';
+import getVerifications from '@/lib/farcaster/getVerifications';
 import { NextRequest } from 'next/server';
 
 export async function GET(req: NextRequest): Promise<Response> {
-  // Retrieve the wallet_address from the query parameters
-  const wallet_address = req.nextUrl.searchParams.get('wallet_address');
+  // Retrieve the fid from the query parameters
+  const fid = req.nextUrl.searchParams.get('fid');
 
-  if (!wallet_address) {
+  if (!fid) {
     return new Response(JSON.stringify({ error: 'wallet_address is required' }), {
       status: 400,
       headers: {
         'Content-Type': 'application/json',
       },
     });
   }
-  const currentBalance = await stack.getPoints(wallet_address);
+  const verifications = await getVerifications(Number(fid));
```

Comment:
Your current changes add an additional API call, slowing down the end user experience.

Did you consider, rather than changing the API param, simply use the `getAddress` function from viem to standardize address inputs?
- Or, is there a different problem these changes are solving to justify the additional `slowdown` from new API calls?